src/core/types/function/index.js

```diff
@@ -8,9 +8,9 @@
  * so it will return the same cached value even when the arguments are
  * different.
  *
- * @param {function(...):T} fn
- * @return {function(...):T}
  * @template T
+ * @param {(function(...any):T)} fn
```

Comment:
is the any an actual any? or should it be `...args: any[]`?

Reply:
Attempted the suggested change and TS is expecting a comma. I think JSDoc-style annotations expect just types, while TS definitions require arg names.

Reply:
Ah, I got confused. Since its a Closure-style `function` its using closure style param style.